StyleCop.Analyzers/StyleCop.Analyzers.Test/MaintainabilityRules/SA1404UnitTests.cs

```diff
@@ -236,7 +235,7 @@ public string Bar()
             }
             catch (NullReferenceException)
             {
-                Assert.Fail("Diagnostic threw NullReferenceException");
+                Assert.True(false, "Diagnostic threw NullReferenceException");
```

Comment:
:question: Any idea why this `try`/`catch` block is here at all? Fixing it is outside the scope of this pull request and should be handled separately (or at minimum, in a commit by itself).